lib/measured/rails/active_record.rb

```diff
@@ -45,8 +45,8 @@ def measured(measured_class, *fields)
             precision = self.column_for_attribute(value_field_name).precision
             scale = self.column_for_attribute(value_field_name).scale
             rounded_to_scale_value = incoming.value.round(scale)
-            ## For BigDecimal#split syntax, refer http://ruby-doc.org/stdlib-2.1.1/libdoc/bigdecimal/rdoc/BigDecimal.html#method-i-split
-            if rounded_to_scale_value.split[1].size > precision
+            
+            if rounded_to_scale_value.to_s.length > precision
```

Comment:
Because if I am reading this correctly we are counting the `.` here in the length.